src/index.js

```diff
@@ -1,13 +1,17 @@
 import './style.css';
-import { getTodos } from './modules/task.js';
+import { allTasks } from './modules/task.js';
+import { displayBox } from './modules/intravtive-list.js';
 
 const addTaskInput = document.querySelector('.add-task__input');
 const taskList = document.querySelector('.add-task');
 const todoItem = JSON.parse(localStorage.getItem('todoItem')) || [];
 const addIcon = document.querySelector('.add__icon');
+const clear = document.querySelector('.clear');
 // eslint-disable-next-line import/prefer-default-export, import/no-mutable-exports
 export let editIndex = null;
```

Comment:
Great job so far, but at line 11 of your index.js file, there is no need to add this, considering the fact that you wouldn't make use of it.